lua/lazyvim/plugins/extras/dap/core.lua

```diff
@@ -69,6 +69,11 @@ return {
       vscode.json_decode = function(str)
         return vim.json.decode(json.json_strip_comments(str))
       end
+
+      -- Extends dap.configurations with entries read from .vscode/launch.json
+      if vim.fn.filereadable(".vscode/launch.json") then
+        vscode.load_launchjs(nil, {})
```

Comment:
```suggestion
        vscode.load_launchjs()
```

This is sortof a nitpick. We can rock the defaults here rather than hardcoding an empty opts. This doesn't change the current behavior but if there are other defaults down the line it would pick them up